src/script_lua.c

```diff
@@ -1504,9 +1505,16 @@ void luaRegisterRedisAPI(lua_State* lua) {
     lua_pushcfunction(lua,luaRedisAclCheckCmdPermissionsCommand);
     lua_settable(lua,-3);
 
-    /* Finally set the table as 'redis' global var. */
+    /* Finally set the table as 'redis' global var. 
+     * We will also alias it to 'server' global var to remove redis from code base. 
+     * In future major version release, we can remove 'redis' global var entirely. */
     lua_setglobal(lua,REDIS_API_NAME);
```

Comment:
Minor preference, I would put the SERVER_API_NAME one first and then add the redis one with a comment explaining it is for compatibility. Let's also drop the mention about dropping it in future releases, I'm not sure we're ever going to do that and would rather not have it in a comment :)

Reply:
sounds good. changing.